lib/protocol/message.go

```diff
@@ -13,13 +13,8 @@ import (
 
 var (
 	sha256OfEmptyBlock        = sha256.Sum256(make([]byte, BlockSize))
-	Version             int32 = 1
-	HelloMessageMaxSize       = 4 + 64 + // DeviceName (4 bytes for length + 64 max length)
-		4 + 64 + // ClientName
-		4 + 64 + // ClientVersion
-		4 + // ProtocolVersion
-		4 + // PayloadType
-		4 + 256 // Payload
+	Version            int32  = 1
```

Comment:
Hmm. We have a version field in the header of every message, and it has a different value (version=0 currently). What's the relation between this version and that?